### PR TITLE
feature/DL-54

### PR DESCRIPTION
**Pull request by wojwarych**
1. Powershell and *nix scripts to clear cache from ```*.pyc``` files (if something goes wrong with running tests on docker)
2. Added logic for ```search()``` func in ```index.py``` in order to get books data (```book.id```, ```book.original_title``` and ```book.authors```) and show it as a sorted list (by last names of authors) of links to books in ```search.html``` -> each link redirect to url with book details via ```book.id```
3. Some changes in stylesheets, added ```search.css``` stylesheet for proper layout of logic
4. Added js script ```search.min.js``` in new folder ```static/js/``` which handles with filtering data in list so that ```SearchForm``` in view is unnecessary
5. Clean up order of imports in ```*.py``` files
6. Tests in progress

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" type="text/css" href="/static/css/base.css">
     <link rel="stylesheet" type="text/css" href="/static/css/home.css">
     <link rel="stylesheet" type="text/css" href="/static/css/authentication.css">
+    <link rel="stylesheet" type="text/css" href="/static/css/search.css">
```
Comment (oskam) [inline templates/base.html]: I was thinking about adding template-specific css links to `{% block head %}`  (direct in html file related with styles , not in `base.html`). But maybe @etude-ist will determine it.

Comment (wojwarych) [reply templates/base.html]: Yeah it seems more reusable and less 'garbage' making.

**Comment by wojwarych** [comment]
I'll resolve this conflicts after approvals of team.
Mainly those are some minor imports conflicts

**Comment by wojwarych** [comment]
I forgot to remove ```print(books)``` in line 146 in ```index.py```
It's freeze time, please remove it during the merge. :)

**Pull request merged**